apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html

```diff
@@ -21,7 +21,8 @@ <h4 class="modal-title bg pull-left">Adjust Payment</h4>
                 <div class="col-md-3 text-right font-14 pt-2 xs-left" *ngIf="!isUpdateMode">
                     <p>Invoice Total: <i class="font-10"></i>{{invoiceFormDetails.accountDetails.currencySymbol}}{{adjustPayment?.grandTotal | giddhCurrency}}
                     </p>
-                    <p>Due <span class="text-light">(Post Adj):</span> <i class="font-10"></i>{{invoiceFormDetails.accountDetails.currencySymbol}}{{ adjustPayment.grandTotal - adjustPayment.totalAdjustedAmount - depositAmount | giddhCurrency}}
+                    <p>Due <span class="text-light">(Post Adj):</span> <i class="font-10"></i> {{invoiceFormDetails.accountDetails.currencySymbol}} {{ adjustPayment.grandTotal - adjustPayment.totalAdjustedAmount - depositAmount + adjustPayment.tcsTotal +
```

Comment:
Avoid expression calculation in HTML as it is run in every change detection even if it is not required. Assign this expression to a variable and update that variable when needed

Reply:
This all variable have different value in different action no need to place or assign in single variable because all these is calculating in different way and actions